### PR TITLE
[device/accton]: Add / update a new supported device and platform, AS7716-32X 

### PR DESCRIPTION
**Pull request by pollyhsu2git**

### CODE REVIEW

**Comment by pollyhsu2git** [comment]
th-as7716-32x100G.config.bcm

File: device/accton/x86_64-accton_as7716_32x-r0/Accton-AS7716-32X/sai.profile [hunk 1]
```diff
@@ -0,0 +1,2 @@
+SAI_INIT_CONFIG_FILE=/etc/bcm/th-as7716-32x100G.config.bcm
```
Comment (stcheng) [inline device/accton/x86_64-accton_as7716_32x-r0/Accton-AS7716-32X/sai.profile]: Could you follow the [Porting Guide](https://github.com/Azure/SONiC/wiki/Porting-Guide) and remove the config.bcm from the pull request and send it directly to me after verifying the file meets the requiremetns?

Comment (pollyhsu2git) [inline device/accton/x86_64-accton_as7716_32x-r0/Accton-AS7716-32X/sai.profile]: @stcheng, Sure!  We put it here just to reminder you that we've came up our config.bcm for you to package it.

Comment (pollyhsu2git) [inline device/accton/x86_64-accton_as7716_32x-r0/Accton-AS7716-32X/sai.profile]: @stcheng, the deletion of th-as7716-32x100G.config.bcm was done and we sent it to via your hotmail e-mail account.

**Comment by lguohan** [comment]
retest this please

**Pull request merged**